src/main/resources/application-deployment.yaml

```diff
@@ -23,6 +23,18 @@ spring:
     show-sql: true
     open-in-view: false
 
+  mail:
+    host: smtp.gmail.com
+    port: 587
+    username: sourcery.graduates@gmail.com
+    password: tvezahniilhaqlga
```

Comment:
Having the SMTP password hardcoded is a critical security risk.
We should store credentials securely using environment variables